hack/changelog-template.txt

```diff
@@ -0,0 +1,31 @@
+%%TITLE%%
+
+%%RELEASE_DATE%%
+
+### <i class="fa-solid fa-bomb"></i> Breaking Changes
+
+### <i class="fa-solid fa-rocket"></i> Features
+
+### <i class="fa-solid fa-bug-slash"></i> Fixes
+
+### <i class="fa-solid fa-box"></i> Helm Chart
+
+### <i class="fa-solid fa-upload"></i> Dependencies
+
+### <i class="fa-solid fa-download"></i> Upgrade
+
+- For NGINX, use the %%IC_VERSION%% images from our
+[DockerHub](https://hub.docker.com/r/nginx/nginx-ingress/tags?page=1&ordering=last_updated&name=%%IC_VERSION%%),
+[GitHub Container](https://github.com/nginxinc/kubernetes-ingress/pkgs/container/kubernetes-ingress),
+[Amazon ECR Public Gallery](https://gallery.ecr.aws/nginx/nginx-ingress) or [Quay.io](https://quay.io/repository/nginx/nginx-ingress).
+- For NGINX Plus, use the %%IC_VERSION%% images from the F5 Container registry,
+the [AWS Marketplace](https://aws.amazon.com/marketplace/search/?CREATOR=741df81b-dfdc-4d36-b8da-945ea66b522c&FULFILLMENT_OPTION_TYPE=CONTAINER&filters=CREATOR%2CFULFILLMENT_OPTION_TYPE),
+the [GCP Marketplace](https://console.cloud.google.com/marketplace/browse?filter=partner:F5,%20Inc.&filter=solution-type:k8s&filter=category:networking)
+or build your own image using the %%IC_VERSION%% source code
+- For Helm, use version %%HELM_CHART_VERSION%% of the chart.
+
+### <i class="fa-solid fa-life-ring"></i> Supported Platforms
+
+We will provide technical support for NGINX Ingress Controller on any Kubernetes platform that is currently supported by
+its provider and that passes the Kubernetes conformance tests. This release was fully tested on the following Kubernetes
+versions: x.xx-x.xx.
```

Comment:
I don't think adding this to every minor release adds much value. Should we just add a link to https://docs.nginx.com/nginx-ingress-controller/technical-specifications/#supported-kubernetes-versions at the top of the page? @brianehlert @ADubhlaoich @Jcahilltorre